getMultimedias.js

```diff
@@ -23,13 +23,16 @@
 
       const [yearElement, runtimeElement] = div.querySelectorAll('.dli-title-metadata-item');
 
-      const year = yearElement.textContent;
-      const runtime = runtimeElement.textContent;
+      const year = yearElement ? yearElement.textContent.trim() : '';
+      const runtime = runtimeElement ? runtimeElement.textContent.trim() : '';
 
       // const genres = getTextContentIfExists(div, ".genre");
+	  const voteCountElement = div.querySelector('.ipc-rating-star--voteCount');
+      const voteCount = voteCountElement ? voteCountElement.textContent.trim() : '';
```

Comment:
Could we get rid of the vote count? I didn't put it intentionally as I would like it to be conditionally rendered as well, but I also don't want to pollute the inputs with the current design. Maybe in the future

Reply:
ok